### ISSUE TITLE
Feedback & possible feature request

### ISSUE THREAD

**Issue by deekmcsquid** [DagAgren/toot#384]
Wasn't sure where to leave feedback on the app. Was told it was the best app for Mastodon on iOS and paid for it and downloaded it yesterday, but found it to be so busy visually that it was overwhelming for my ADHD eyes/brain (e.g. added visual decor elements such as the stripes across the top of CW toots). It's like the app is shouting at me around the already busy amounts of text, avatars, media posts, etc. 

Knowing that there was no way I could ever use this app, I closed it immediately and requested a refund on it. If I knew there was a way to go into a more minimalist mode or if I knew such a mode was coming up, I would happily buy it again.

**Comment by stillnotstars** [comment]
i'd love a mode with reduced animations as well!

**Comment by ChlorideCull** [comment]
Yeah, I really can't use it until there's a way to make it less busy, things that could be removed off the top of my head:
- warning stripes on CWs
- animation on favorite/boost
- favorited toots becoming shiny and a different color (I'd prefer it if you changed the color of the button label instead, still visible but less distracting)

**Comment by RobATX** [comment]
Yes, please. While they are fun, then tend to slow down the ability to take actions quickly.
